packages/flutter/lib/src/gestures/events.dart

```diff
@@ -86,6 +130,44 @@ int nthMouseButton(int number) => (kPrimaryMouseButton << (number - 1)) & kMaxUn
 /// for some stylus buttons.
 int nthStylusButton(int number) => (kPrimaryStylusButton << (number - 1)) & kMaxUnsignedSMI;
 
+/// Returns the button of `buttons` with the smallest integer.
+///
+/// A `buttons` is a bitfield where each set bit represents a button. This
+/// function returns the set bit closest to the least significant bit.
+///
+/// It returns 0 for 0.
```

Comment:
Maybe a more verbose explanation is clearer?:
```suggestion
/// When [buttons] is zero, [smallestButton] returns zero.
```